boards/arm/bl5340_dvk/bl5340_dvk_cpuapp_common.dts

```diff
@@ -65,6 +65,14 @@
 		};
 	};
 
+	gpio_fwd: nrf-gpio-forwarder {
+		compatible = "nordic,nrf-gpio-forwarder";
+		status = "okay";
+		uart {
+			gpios = <&gpio1 8 1>, <&gpio1 10 1>, <&gpio0 7 1>, <&gpio0 9 1>;
```

Comment:
These should all be p1.xx:
p1.07 (RTS)
p1.09 (CTS)
p1.08 (TX)
p1.10 (RX)

Not sure what the 1 at the end is for, but it's 0 in the nRF5340.

Reply:
My mistake, I thought the second number was the port, but it is something else.